src/Native/Rails.js

```diff
@@ -15,9 +14,10 @@ var _NoRedInk$elm_rails$Native_Rails = function(){
       // ignore document-based errors
     }
 
-    if ((csrfTokenNode !== null) && (typeof csrfTokenNode.content === "string")){
-      csrfToken = { ctor: 'Just', _0: csrfTokenNode.content };
-    }
+    var csrfToken =
+      ((csrfTokenNode !== null) && (typeof csrfTokenNode.content === "string"))
+        ? { ctor: 'Ok', _0: csrfTokenNode.content }
+        : { ctor: 'Err', _0: "<meta name=\"csrf-token\"> was not found in document.head." };
```

Comment:
I think terninary operators are really hard to read - I'd prefer the more verbose if..else here, because it'll make it easier to spot errors. but :+1:  for making it a result